spec/acceptance/shared/stubbing_requests.rb

```diff
@@ -593,6 +593,23 @@ def stub_non_globally
           end
         end
       end
+
+      context "when global stub should be invoked last" do
+        before do
+          WebMock.globally_stub_request(:after_stubs) do
+            { body: "global stub body" }
+          end
+        end
+
+        it "uses global stub when non global stub is defined" do
```

Comment:
I guess that should be `when non-global stub is not defined`?

Reply:
Yeap - fixed.